Amdad_home.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Amdad Homepage</title>
+    <link rel="stylesheet" href="home_style.css" >
+</head>
+<body>
+    <!-- Button group for navigation though website-->
+    <div class="btn-group">
```

Comment:
Semantic HTML is great for making clean, readable code as well as code that is accessible to all users. Read up on semantics and how you can implement them into your code rather than storing code within divs. Its best practice for our HTML files to be semantic.